grakn-engine/src/main/java/ai/grakn/engine/user/SystemKeyspaceUsers.java

```diff
@@ -49,6 +49,12 @@
  */
 public class SystemKeyspaceUsers extends UsersHandler {
     private final Logger LOG = LoggerFactory.getLogger(SystemKeyspaceUsers.class);
+    private final EngineGraknGraphFactory factory;
+
+    SystemKeyspaceUsers(String adminPassword, EngineGraknGraphFactory factory) {
```

Comment:
I don't like that sometimes we pass in this `EngineGraknGraphFactory` and other times we don't.